packages/listview/src/components/ColumnHeaderFilterText/FilterText/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { RenderIf, ButtonIcon, Button } from 'react-rainbow-components';
+import { TrashFilled } from '@rainbow-modules/icons';
+import {
+    StyledContentField,
+    StyledOr,
+    StyledInput,
+    StyledButton,
+    StyledFooter,
+    StyledIconPlus,
+    StyledContent,
+} from './styled';
+import formatFilters from './helpers/formatFilters';
+
+function FilterText(props) {
+    const { onFilter, defaultFilters, headerText, onRequestClose } = props;
+    const [filters, setFilters] = useState(formatFilters(defaultFilters));
```

Comment:
here this state is the same in the parent component, can we keep only one from the parent?